### PR TITLE
feat(query): Added ETCD TLS Certificate Files Not Properly Set for Kubernetes

### PR DESCRIPTION
**Pull request by cxMiguelSilva**
**Proposed Changes**
- Added ETCD TLS Certificate Files Not Properly Set for Kubernetes

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/k8s/etcd_tls_certificate_files_not_properly_set/query.rego [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	specInfo := k8sLib.getSpecInfo(resource)
+	types := {"initContainers", "containers"}
+	container := specInfo.spec[types[x]][j]
+	common_lib.inArray(container.command, "etcd")
+	not k8sLib.startWithFlag(container, "--cert-file")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("metadata.name={{%s}}.%s.%s.name={{%s}}.command", [metadata.name, specInfo.path, types[x], container.name]),
+		"issueType": "IncorrectValue",
```
Comment (rafaela-soares) [inline assets/queries/k8s/etcd_tls_certificate_files_not_properly_set/query.rego:18]:
```suggestion
        "issueType": "MissingAttribute",
```

File: assets/queries/k8s/etcd_tls_certificate_files_not_properly_set/query.rego [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	specInfo := k8sLib.getSpecInfo(resource)
+	types := {"initContainers", "containers"}
+	container := specInfo.spec[types[x]][j]
+	common_lib.inArray(container.command, "etcd")
+	not k8sLib.startWithFlag(container, "--key-file")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("metadata.name={{%s}}.%s.%s.name={{%s}}.command", [metadata.name, specInfo.path, types[x], container.name]),
+		"issueType": "IncorrectValue",
```
Comment (rafaela-soares) [inline assets/queries/k8s/etcd_tls_certificate_files_not_properly_set/query.rego:37]:
```suggestion
        "issueType": "MissingAttribute",
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/etcd_tls_certificate_files_not_properly_set/query.rego [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("metadata.name={{%s}}.%s.%s.name={{%s}}.command", [metadata.name, specInfo.path, types[x], container.name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": "--key-file flag should be defined",
+		"keyActualValue": "--key-file flag is not defined",
+		"searchLine": common_lib.build_search_line(split(specInfo.path, "."), [types[x], j, "command"]),
+	}
+}
```
Comment (rafaela-soares) [inline assets/queries/k8s/etcd_tls_certificate_files_not_properly_set/query.rego:42]: We could join these policies

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Review by joaorufi — APPROVED** [review]

**Pull request merged**